.gitignore

```diff
@@ -149,5 +149,61 @@ docs/generated
 docs/api
 docs/whatsnew/latest_changelog.txt
 ndcube/version.py
-htmlcov/
+#htmlcov/
```

Comment:
Presumably the changes in this file are to do with your local setup?  Should probably also be removed for this PR.

Reply:
Yeah, probably. But I think I will need these in the "master" `.gitigore` file for it to work though. Let me double check and follow up accordingly.

Reply:
Think we will need to keep it if I am working with PyCharm... Have removed the duplicated entry though.